torchci/clickhouse_queries/tritonbench_benchmark/query.sql

```diff
@@ -0,0 +1,47 @@
+WITH results AS (
```

Comment:
As this is a brand new dashboard, it's probably ok to keep these separate queries for flexibility.  But I suspect that we can fold them into existing https://github.com/pytorch/test-infra/tree/main/torchci/clickhouse_queries/oss_ci_benchmark_llms.  That's for a later PR if needed

cc @yangw-dev

Reply:
Right, we are open to converge into the existing queries if that is possible.